README.md

```diff
@@ -60,7 +60,7 @@ edition = "2021"
 crate-type = [
     "staticlib",  # Ensure it gets compiled as a (static) C library
   # "cdylib",     # If you want a shared/dynamic C library (advanced)
-    "lib",        # For downstream Rust dependents: `examples/`, `tests/` etc.
```

Comment:
Nice to get it properly documented 🙏 although I do like the mention of downstream rust dependents as well, so I'd like to keep it.

What about:

```python
    "lib",        # For `generate-headers` and other downstream rust dependents
                  # such as integration `tests/`, doctests, and `examples/`
```

Reply:
> Nice to get it properly documented 🙏 although I do like the mention of downstream rust dependents as well, so I'd like to keep it.
> 
> What about:
> 
> ```python
>     "lib",        # For `generate-headers` and other downstream rust dependents
>                   # such as integration `tests/`, doctests, and `examples/`
> ```

Sure, took your suggestion verbatim.